docs/docsite/rst/user_guide/vars_list.rst

```diff
@@ -0,0 +1,110 @@
+Special Variables
+=================
+
+Magic
+-----
+These variables are directly not settable by the user, Ansible will always override them to reflect internal state.
+
+ansible_check_mode
+    Boolean that indicates if we are in check mode or not
+
+ansible_diff_mode
+    Boolean that indicates if we are in diff mode or not
+
+ansible_forks
+    Integer reflecting the number of maximum forks available to this run
+
+ansible_inventory_sources
+    List of sources used as inventory
+
+ansible_play_batch
+    List of active hosts in the current play run limited by the serial, aka 'batch'. Failed/Unreachable hosts are not considered 'active'.
+
+ansible_play_hosts
+    The same as ansible_play_batch
+
+ansible_play_hosts_all
+    List of all the hosts that were targeted by the play
+
+ansible_playbook_python
+    The path to the python interpreter being used by Ansible on the controller
+
+ansible_serach_path
+    Current search path for action plugins and lookups, i.e where we search for relative paths when you do ``template: src=myfile``
+
+ansible_verbosity
+    Current verbosity setting for Ansible
+
+ansible_version
+   Dictionary/map that contains information about the current running version of ansible, it has the following keys: full, major, minor, revision and string.
+
+group_names
+    List of groups the current host is part of
+
+groups
+    A dictionary/map with all the groups in inventory and each group has the list of hosts that belong to it
+
+hostvars
+    A dictionary/map with all the hosts in inventory and variables assigned to them
+
+inventory_hostname
+    The inventory name for the 'current' host being iterated over in the play
+
+inventory_hostname_short
+    The short version of `inventory_hostname`
+
+inventory_dir
+    The directory of the inventory source in which the `inventory_hostname` was first defined
+
+inventory_file
+    The file name of the inventory source in which the `inventory_hostname` was first defined
+
+omit
+    Special variable that allows you to 'omit' an option in a task, i.e ``- user: name=bob home={{ bobs_home|default(omit)}}``
+
+play_hosts
+    Deprecated, the same as ansbile_play_batch
+
+playbook_dir
```

Comment:
Would be great to have a `playbook_name` to get the filename of the running playbook.  Couldn't find the equivalent functionality

Reply:
this pr is documenting existing feature, not for adding new ones, open a feature request for that